apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -118,7 +118,7 @@ export class FilesStorageUC {
 			await this.fileRecordRepo.save(entity);
 			const filePath = this.createPath(params.schoolId, entity.id);
 			await this.storageClient.create(filePath, fileDescription);
-			await this.antivirusService.send(entity);
+			this.antivirusService.send(entity);
```

Comment:
It send and how the error is come to this place without await? Is this handled by the consumer/producer it self and throw up? Is this catch by the interceptor error pipline to log this error?

Reply:
https://github.com/golevelup/nestjs/tree/master/packages/rabbitmq#publising-messages-fire-and-forget